pkg/cloud/cloud.go

```diff
@@ -168,10 +185,16 @@ func (c *cloud) DescribeFileSystem(ctx context.Context, fileSystemId string) (*F
 		return nil, err
 	}
 
+	mountName := "fsx"
+	if fs.LustreConfiguration.MountName != nil {
+		mountName = *fs.LustreConfiguration.MountName
+	}
+
 	return &FileSystem{
 		FileSystemId: *fs.FileSystemId,
 		CapacityGiB:  *fs.StorageCapacity,
 		DnsName:      *fs.DNSName,
+		MountName:    mountName,
```

Comment:
Can you just set `*fs.LustreConfiguration.MountName` here?

Reply:
It is better to add the nil check here just in case.